Units/fabricaheks.lua

```diff
@@ -46,7 +46,7 @@ local unitDef = {
 	-- Building
 	
 	Builder = true,
-	canMove = true,

```

Comment:
Ok, I'll do it when I get home.



> On Jun 19, 2016, at 1:37 PM, TurBoss <notifications@github.com> wrote:
> 
> In Units/fabricaheks.lua:
> 
> > @@ -46,7 +46,7 @@ local unitDef = {
> >  	-- Building
> >  	
> >  	Builder = true,
> > -	canMove = true,
> revert this and i'll merge
> this is neede to set a destination point for units begin created
> 
> ―
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>